### ISSUE TITLE
Protection stone placement spell casting time

### ISSUE THREAD

**Issue by jamesol1** [espidev/ProtectionStones#287]
I was planning on using protection stones as a part of a nomad style protection system. I think this would be a nice option for others too. Is having a spell cast time for the protectionstones to activate. Or perhaps adding this to /ps toggle.

That way players can't instantly protect a region and effectivelty abuse it.

Thanks for the amazing plugin

**Comment by espidev** [comment]
I'm not sure what spell casting refers to... You mean a delayed creation time for regions?

**Comment by jamesol1** [comment]
> I'm not sure what spell casting refers to... You mean a delayed creation time for regions?

Yep that’s exactly it. A configurable time until the protection stone is activated or properly placed.
